rfcs/XXX-eresource_authentication/README.md

```diff
@@ -0,0 +1,59 @@
+# E-Resource Authentication
+
+Wellcome Collection like many other institutions offers its staff and patrons access to electronic journals & books offered by online databases/publishers.
+
+The list of databases offered is [available on the Wellcome Collection site](https://wellcomecollection.org/pages/YDaP2BMAACUAT7DS).
+
+The "Collections and Research" team handles relationships with publishers / database providers, negotiates terms of access and communicate changes in access requirements.
+
+As part of decommissioning the Wellcome Library site we must ensure continuity of access to those resources for patrons and staff.
+
+## E-resources in the Catalogue API
+
+E-resources are currently available from the page linked above, but [can also be found](https://api.wellcomecollection.org/catalogue/v2/works?query=awa6c6gm&include=items) in the catalogue API.
+
+Items will have an `online-resource` location:
+
+![online resource item](online_resource.png)
```

Comment:
nit: can you make this JSON? Easier to read in Markdown, search for, more accessible than an image.